estuary_updater/handlers/koji.py

```diff
@@ -64,6 +65,28 @@ def build_complete_handler(self, msg):
 
             build = self.get_or_create_build(msg['body']['msg']['info']['id'])
 
+            if build.__label__ == ModuleKojiBuild.__label__:
+                extra_json = msg['body']['msg']['info']['extra']
+                module_extra_info = extra_json.get('typeinfo', {}).get('module')
+                module_build_tag_name = module_extra_info.get('content_koji_tag')
+                if module_build_tag_name:
```

Comment:
Optional: You can assume this is always set if the build is a module, so you can dedent this whole block of code by removing this if statement. It doesn't hurt to keep this though.